src/darwin/Framework/chip_xcode_build_connector.sh

```diff
@@ -78,7 +83,7 @@ declare -a args=(
     'target_cpu="'"$ARCHS"'"'
     'target_defines='"$target_defines"
     'default_configs_cosmetic=[]'
-    'target_cflags=["-target","'"$ARCHS"'-'"$LLVM_TARGET_TRIPLE_VENDOR"'-'"$LLVM_TARGET_TRIPLE_OS_VERSION"'", "-DNDEBUG"]'
+    'target_cflags=["-target","'"$ARCHS"'-'"$LLVM_TARGET_TRIPLE_VENDOR"'-'"$LLVM_TARGET_TRIPLE_OS_VERSION"'", "-DNDEBUG", '"$bitcode"']'
```

Comment:
glad this works

Reply:
At least as far as I can tell :). Verified with the simulator and device builds (they use different Bitcode settings apparently)